cabot/cabotapp/tasks.py

```diff
@@ -89,11 +89,11 @@ def clean_db(days_to_retain=60):
     from .models import StatusCheckResult, ServiceStatusSnapshot
     from datetime import timedelta
 
-    to_discard_results = StatusCheckResult.objects.filter(time__lte=timezone.now()-timedelta(days=days_to_retain))
-    to_discard_snapshots = ServiceStatusSnapshot.objects.filter(time__lte=timezone.now()-timedelta(days=days_to_retain))
+    to_discard_results = StatusCheckResult.objects.filter(time_complete__lte=timezone.now() - timedelta(days=days_to_retain))
```

Comment:
Are there ever cases where time_complete is not set? Just asking because it appears to be nullable where as time isn't. Looks like it's dealt with in StatusCheck run so should be fine.